StudentManagement/StudentManagement.Console/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿using StudentManagement.Console;
+
+List<Student> students = new List<Student>();
+students.Add(new Student());
```

Comment:
instead of accessing list by using [0] index - you can create student first and then add it to the student list in the end of the file

Reply:
good idea, i will use that. thank you